tasks/root-password.yml

```diff
@@ -7,7 +7,8 @@
       Warning!! the MariaDB root password was left empty. Please set a custom
       password with role variable mariadb_root_password to secure your database
       server!
-  when: mariadb_root_password | length == 0
+#  when: mariadb_root_password | length == 0   ---It is not Vault compatible
+  when: mariadb_root_password == ''
```

Comment:
In the current version, this check is implemented as `when: not mariadb_root_password`. Would that be Vault compatible?